ngrinder-controller/src/main/java/org/ngrinder/infra/webhook/plugin/NgrinderWebhookPlugin.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2012-present NAVER Corp.
+ *
+ * This file is part of The nGrinder software distribution. Refer to
+ * the file LICENSE which is part of The nGrinder distribution for
+ * licensing details. The nGrinder distribution is available on the
+ * Internet at https://naver.github.io/ngrinder
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ngrinder.infra.webhook.plugin;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.ngrinder.extension.OnTestLifeCycleRunnable;
+import org.ngrinder.infra.webhook.model.Event;
+import org.ngrinder.infra.webhook.model.WebhookConfig;
+import org.ngrinder.infra.webhook.service.WebhookConfigService;
+import org.ngrinder.infra.webhook.service.WebhookService;
+import org.ngrinder.model.PerfTest;
+import org.ngrinder.service.IPerfTestService;
+
+import static org.ngrinder.infra.webhook.model.Event.FINISH;
+import static org.ngrinder.infra.webhook.model.Event.START;
+
+/**
+ * nGrinder webhook plugin.
+ *
+ * @since 3.5.2
+ */
+@SuppressWarnings("DuplicatedCode")
+@Slf4j
+@AllArgsConstructor
+public class NgrinderWebhookPlugin implements OnTestLifeCycleRunnable {

```

Comment:
We mostly named a class like `NGrinderSomething` not `NgrinderSomething`. It would be better to change class name to `NGrinderWebhookPlugin` and `NGrinderWebhookClient`.

Reply:
fixed

https://github.com/naver/ngrinder/blob/447a3f6a99ed5c8a5f5634deca0be9662b5f5f1f/ngrinder-controller/src/main/java/org/ngrinder/infra/webhook/plugin/NGrinderWebhookPlugin.java#L44

https://github.com/naver/ngrinder/blob/447a3f6a99ed5c8a5f5634deca0be9662b5f5f1f/ngrinder-controller/src/main/java/org/ngrinder/infra/webhook/service/NGrinderWebhookClient.java#L42